service-base/src/main/java/org/eclipse/hono/service/monitoring/LoggingConnectionEventProducer.java

```diff
@@ -12,76 +12,27 @@
  */
 package org.eclipse.hono.service.monitoring;
 
-import static java.lang.invoke.MethodType.methodType;
-
-import java.lang.invoke.MethodHandle;
-import java.lang.invoke.MethodHandles;
-
 import org.eclipse.hono.service.auth.device.Device;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.slf4j.event.Level;
 
 /**
  * A logging only implementation.
+ * <p>
+ * This implementation only create log messages as "events". It will log everything on "info" level.
```

Comment:
In most (if not all) other JavaDoc we use `<em>` to highlight or emphasize terms and use `@code` to mark literals instead of using quotes. Can you please adapt accordingly?

So, FMPOV thsi should read:
> This implementation only create log messages as <em>events</em>. It will log everything at {@code info} level.

Reply:
Done